### ISSUE TITLE
chromosomes are everywhere- should they be?

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/microservices#345]
@alancleary just noticed that I inherited something from your setup.cfg that doesn't seem to make a lot of sense:
```
[options.entry_points]
console_scripts =
    chromosome = search.__main__:main
```
it probably doesn't actually matter, but the occurrence of "chromosome" in this and all other microservices setup.cfg files that are not in fact the chromosome microservice seems like it's probably a minor goof

**Comment by alancleary** [comment]
Good catch! Clearly we're not using the CLI support that this adds to each microservice. It's still worth fixing, though. Perhaps it's worth putting a test in place to make sure this is right...
